website/source/docs/platform/k8s/helm.html.md

```diff
@@ -377,6 +377,176 @@ to run the sync program.
             "envoy_dogstatsd_url": "udp://127.0.0.1:9125"
           }
         ```
+* <a name="v-meshgateway" href="#v-meshgateway">`meshGateway` <sup>(beta)</sup></a> - Values that configure Connect [Mesh Gateways](/docs/connect/mesh_gateway.html).
+
+  * <a name="v-meshgateway-enabled" href="#v-meshgateway-enabled">`enabled`</a> (`boolean: false`) - If true, gateways will be created and
+    Consul servers will be configured to support them. If setting this in an already running cluster, you'll need to
```

Comment:
This sounds a bit like setting enabled to true will take care of the prereqs. Might be clearer if we put this sentence in the active voice. Also, I'm not 100% sure about what exactly "gateways will be created" means. Does this schedule, configure, and manage envoy?

Reply:
What about:
```
Setting to true will cause mesh gateways to be deployed.
```

Reply:
> Does this schedule, configure, and manage envoy?

Yes it creates a Deployment with Pods that run envoy and register as gateways.